src/Client.php

```diff
@@ -509,7 +510,15 @@ public function takeScreenshot($saveAs = null)
     {
         $this->start();
 
-        return $this->webDriver->takeScreenshot($saveAs);
+        if (null === $saveAs || (new Filesystem())->isAbsolutePath($saveAs)) {
```

Comment:
I mean this one: why would we force one to pass an absolute path? I don't understand.